docker-compose.yml

```diff
@@ -0,0 +1,23 @@
+version: '3'
+services:
+
+  polkadot:
+    image: chevdor/polkadot:latest
```

Comment:
why aren't we using official image?
Pleaase use speicifc tag instead of latest

Reply:
I replaced it with official image